### ISSUE TITLE
Cannot publish section

### ISSUE THREAD

**Issue by philipstarkey** [varunsrin/one-py#2]
Hi,

I've tried to publish a section from OneNote 2013 to PDF, using your library (`ONProcess.publish()` method) but it failed with an exception. I haven't tried publishing anything else, so it probably isn't just section specific, and I haven't tried with OneNote 2010. In case it matters, the pywin32 version is 219 (the latest).

It seems that the call to `ONProcess.process.Publish(hierarchy_id, target_file_path, publish_format, clsid_of_exporter)` fails with the default argument of clsid_of_exporter (currently `0`). If I don't use your convenience method and instead make the call myself, and omit the `clsid_of_exporter` argument, then it publishes successfully. E.g. `self.process.Publish(hierarchy_id, target_file_path, publish_format)` works. 

I initially suspected the problem was because the OneNote API [documentation](https://msdn.microsoft.com/en-us/library/office/jj680120%28v=office.15%29.aspx#ON14DevRef_Application_PageContent) says the type of the `clsid_of_exporter` argument should be a string (and you are using an integer). However, I tried a string and it still failed. I don't seem to be able to set the `clsid_of_exporter` argument to anything that works.

Fortunately, you can omit that parameter as long as you don't need to export in the emf format (which I don't). If you can't figure out what the default argument should be, it might be a good idea to only pass that parameter to the COM call when it is something other than the default value (so that the method is at least mostly usable!), eg:

``` python
if clsid_of_exporter == 0:
    self.process.Publish(hierarchy_id, target_file_path, publish_format)
else:
    self.process.Publish(hierarchy_id, target_file_path, publish_format, clsid_of_exporter)
```

Cheers,

**Comment by varunsrin** [comment]
Hmm, the problem is because the API expects a BSTR (binary string), while Python is providing a Unicode string. I'll dig into this soon, thanks for the detailed report.

**Comment by varunsrin** [comment]
I should have looked at the docs a little closer, all strings are converted to the right BSTR class by the interop. 

It turns out that passing an empty string in Python sends the right default value to the COM interface. I'm not sure if its an error in the documentation, or an idiosyncrasy of the interop, but it works with the latest commit.

**Comment by varunsrin** [comment]
I just published a new v0.2.1 with the fixes, you should be able to pip install the latest version to work around these issues.
